modules/nativescript/nativescript.cpp

```diff
@@ -407,6 +412,24 @@ Variant NativeScript::_new(const Variant **p_args, int p_argcount, Variant::Call
 		return Variant();
 	}
 
+	call("_init", p_args, p_argcount, r_error);
+
+	if (r_error.error != Variant::CallError::CALL_OK) {
```

Comment:
It should, because then something's not right. `_init` is defined in Object, so it should definitely be there. This only throws errors when something *else* is not right. I tested it, it works fine if `_init` is not defined ;)